scap/oscap/scan.sls

```diff
@@ -13,7 +13,7 @@ create oscap output directory:
 
 run oscap scan:
   cmd.run:
-    - name: '(oscap xccdf eval --profile {{ oscap.profile }} --report {{ oscap.report }} --results {{ oscap.results }} {% if not oscap.scap1_2 -%} --cpe {{ oscap.cpe }} {%- endif %} {{ oscap.xccdf }}) || true'
+    - name: '(oscap xccdf eval --profile {%- if oscap.os == 'rhel' %} profile_{{ oscap.profile }} {%- else %} {{ oscap.profile }} {%- endif %} --report {{ oscap.report }} --results {{ oscap.results }} {% if not oscap.scap1_2 -%} --cpe {{ oscap.cpe }} {%- endif %} {{ oscap.xccdf }}) || true'
```

Comment:
This _could_ be done on the pillar side of things, in the salt-content... Not sure we want to always force-override what the user sends us, here in the formula. Thoughts?

Reply:
Yeah, doing it in pillar is probably better.  I'll give that a try.